fire/cli/mtl.py

```diff
@@ -25,7 +29,41 @@
 from fire.cli import firedb
 
 # Typingelementer fra databaseAPIet.
-from fire.api.model import Punkt, Koordinat, PunktInformation, PunktInformationType
+from fire.api.model import (
+    GeometriObjekt,
+    Point,
+    Punkt,
+    Koordinat,
+    PunktInformation,
+    PunktInformationType,
+    Sag,
+    Sagsevent,
+    SagseventInfo,
+    Sagsinfo,
+)
+
+
+class Koordinatvrider:
+    """Check op på placeringskoordinaterne.
+    Hvis nogle ligner UTM, så regner vi om til geografiske koordinater.
+    NaN og 0 flyttes ud i Kattegat, så man kan få øje på dem
+    """
+
+    utm32 = Proj("proj=utm zone=32 ellps=GRS80", preserve_units=False)
+    assert utm32 is not None, "Kan ikke initialisere projektionselelement utm32"
+
+    def vrid(self, λ, φ):
```

Comment:
Her foretrækker jeg også et mere sigende navn, fx `transformer`. Det ved jeg med det samme hvad betyder, jeg skal slå op i dokumentationen for at forstå hvad "vrid" betyder i denne kontekst.

Derudover, hvilke datatyper er lambda og phi her? floats? Eller noget fancy fra pandas?